include/swift/SILOptimizer/Analysis/ARCAnalysis.h

```diff
@@ -221,6 +221,18 @@ class ConsumedArgToEpilogueReleaseMatcher {
 
   bool hasBlock() const { return HasBlock; }
 
+  bool isEpilogueRelease(SILInstruction *I) const {
```

Comment:
Or keep a SILBasicBlock *ThisBlock to represent the processed block by this ConsumedArgToEpilogueReleaseMatcher. And then we can remove the HasBlock variable.